cmd/image-builder/image-builder.md

```diff
@@ -174,3 +174,29 @@ You can use Image Builder to build images that require access to Go SAP internal
+  supported when running in the GitHub Actions workflow.
+
+### Required Environment Variables
+
+The Jenkins integration relies on a defined contract for sourcing input data via specific environment variables:
+
+| Variable          | Description                                                                                                                                                                                                                                              | Required                                   |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------|
+| `CHANGE_ID`       | Detects if the pipeline is executed for a pull request. Must contain the PR number.                                                                                                                                                                      | Required for pull requests                 |
+| `GIT_URL`         | Repository URL from which an image will be built. Must match the regex defined in [`extractOwnerAndRepoFromGitURL` function](https://github.com/kyma-project/test-infra/blob/df945b96654d60f82b9738cd98129191c5e753c8/cmd/image-builder/config.go#L123). | Always required                            |
+| `GIT_COMMIT`      | Commit SHA: PR HEAD SHA for pull requests, branch HEAD SHA for push events.                                                                                                                                                                              | Always required                            |
+| `CHANGE_BRANCH`   | Name of the pull request's base branch.                                                                                                                                                                                                                  | Required for pull requests                 |
+| `CHANGE_BASE_SHA` | Base branch commit SHA for the pull request.                                                                                                                                                                                                             | Required for pull requests                 |
+| `JENKINS_HOME`    | Indicates Jenkins runtime environment. Presence of this variable signals the tool to run in Jenkins mode (value not tested).                                                                                                                             | Always required for Jenkins pipeline usage |
+
+**Note**: This Jenkins integration is intended exclusively for the SRE team. It should not be used by other teams or projects.
```

Comment:
```suggestion
> [!NOTE]
> This Jenkins integration is intended exclusively for the SRE team. Other teams or projects must not use it.
```

Reply:
Why not put this note at the very beginning of the Jenkins Integration section? It would definitely attract more attention there.

Reply:
Moved to the section beginning.